runtime/themes/everblush.toml

```diff
@@ -0,0 +1,132 @@
+"float" = { fg = "black-light" }
+"function" = { fg = "red" }
+"function.builtin" = { fg = "cyan-light" }
+"function.macro" = { fg = "green" }
+"include" = { fg = "red-light" }
+"keyword" = { fg = "blue" }
+"keyword.function" = { fg = "blue" }
+"keyword.operator" = { fg = "blue-light" }
+"keyword.return" = { fg = "blue" }
+"label" = { fg = "blue" }
+"method" = { fg = "blue-light" }
+"namespace" = { fg = "red-light" }
+"number" = { fg = "yellow" }
+"operator" = { fg = "white" }
+"parameter" = { fg = "red" }
+"parameter.reference" = { fg = "red-light" }
+"property" = { fg = "red" }
+"punctuation.bracket" = { fg = "white" }
+"punctuation.delimiter" = { fg = "white" }
+"punctuation.special" = { fg = "white" }
+"repeat" = { fg = "yellow-light" }
+"string" = { fg = "green" }
+"string.escape" = { fg = "blue" }
+"string.regex" = { fg = "green" }
+"string.special" = { fg = "blue" }
+"symbol" = { fg = "red" }
+"tag" = { fg = "blue" }
+"tag.attribute" = { fg = "red" }
+"tag.delimiter" = { fg = "white" }
+"text" = { fg = "white" }
+"text.danger" = { fg = "black-light" }
+"text.emphasis" = { fg = "white", italic = true }
+"text.environment.name" = { fg = "yellow" }
+"text.environtment" = { fg = "magenta" }
+"text.literal" = { fg = "green", italic = true }
+"text.math" = { fg = "cyan" }
+"text.note" = { fg = "black-light" }
+"text.reference" = { fg = "cyan" }
+"text.strike" = { fg = "white", strikethrough = true }
+"text.strong" = { fg = "white", bold = true }
+"text.title" = { fg = "yellow", bold = true }
+"text.underline" = { fg = "magenta", underline = true }
+"text.uri" = { fg = "yellow", underline = true }
+"text.warning" = { fg = "black", bg = "red" }
+"type" = { fg = "yellow" }
+"type.builtin" = { fg = "yellow" }
+"variable" = { fg = "white" }
+"variable.builtin" = { fg = "blue" }
+
+"diff.plus" = { fg = "blue" }
+"diff.delta" = { fg = "magenta" }
+"diff.minus" = { fg = "red" }
+
+"ui.background" = { fg = "foreground", bg = "background" }
+"ui.cursor" = { modifiers = ["reversed"] }
+#"ui.cursor.match" = { fg = "orange", modifiers = ["bold"] }
+# "ui.cursor.primary" = { modifiers = ["reversed"] }
+"ui.cursorline.primary" = { bg = "cursorline" }
+"ui.help" = { fg = "foreground", bg = "contrast" }
+"ui.linenr" = { fg = "comment" }
+"ui.linenr.selected" = { fg = "foreground" }
+"ui.menu" = { fg = "foreground", bg = "contrast" }
+"ui.menu.selected" = { bg = "black" }
+"ui.popup" = { fg = "foreground", bg = "contrast" }
+"ui.selection" = { bg = "black" }
+"ui.selection.primary" = { bg = "black" }
+"ui.statusline" = { fg = "foreground", bg = "backound" }
+"ui.statusline.inactive" = { fg = "foreground", bg = "backgoround" }
+"ui.statusline.normal" = { fg = "white", bg = "background" }
+"ui.statusline.insert" = { fg = "blue", bg = "background" }
+"ui.statusline.select" = { fg = "cyan", bg = "magenta" }
+"ui.text" = { fg = "foreground" }
+"ui.text.focus" = { fg = "blue" }
+"ui.virtual.ruler" = { bg = "comment" }
+"ui.virtual.whitespace" = { fg = "comment" }
```

Comment:
There are some other `ui.virtual` scopes that should be styled for features like soft-wrap and indentation guides: https://docs.helix-editor.com/master/themes.html#interface

You might be able to re-use the `comment` highlight for everything but the rulers depending on how that looks to you:

```toml
"ui.virtual.ruler" = { bg = "comment" }
"ui.virtual" = { fg = "comment" }
```

Reply:
Thanks for pointing out. I have implemented warp and indent guide. Also set the ruler to a bit different color as it maybe confusing along with other virtual contents that may be in the near the ruler. Thanks!!